global.json

```diff
@@ -1570,7 +1570,7 @@
             "type": "cloud_nodejs",
             "repoUrl": "git://github.com/feedhenry-raincatcher/raincatcher-demo-auth.git",
             "githubUrl": "https://github.com/feedhenry-raincatcher/raincatcher-demo-auth.git",
-            "repoBranch": "refs/tags/release-3.18.0-rc3",
+            "repoBranch": "refs/tags/v2.2.5",
             "category": "Custom Service",
             "forcedSelection": true
           }
```

Comment:
@kpiwko I looked at this again and just updating these like this here won't actually do what we want since the FH-3.18 branch on all the wfm repos was created at the wrong point since master wasn't updated correctly.

I have changed all the FH-3.18 branches and tags on these repos manually:

* https://github.com/feedhenry-raincatcher/raincatcher-demo-auth/tree/FH-v3.18
* https://github.com/feedhenry-raincatcher/raincatcher-demo-cloud/tree/FH-v3.18
* https://github.com/feedhenry-raincatcher/raincatcher-demo-mobile/tree/FH-v3.18
* https://github.com/feedhenry-raincatcher/raincatcher-demo-portal/tree/FH-v3.18

I'm going to remove this change from this PR as it's not going to do anything since the pipeline will just tag the repo at the HEAD of FH-v3.18 and update this anyway.